features/features.go

```diff
@@ -541,7 +541,7 @@ var (
 				reportProblemsToJiraComponent("Image Registry").
 				contactPerson("flavianmissi").
 				productScope(ocpSpecific).
-				enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
+				enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```

Comment:
Was this intended? I think it's why integration tests are failing

Reply:
No. Seems like a rebase issue from my end. Updated.